haproxy/default_frontend.cfg

```diff
@@ -19,6 +40,9 @@ defaults
   timeout server 360000000
   timeout connect 1500000
   timeout http-keep-alive 5m
+
+  errorfiles json
+
```

Comment:
If I understand this correctly, this will always respond with JSON regardless of the Accepts header?

Reply:
Correct, since HAProxy is not being called directly by browsers or mobile apps adding a conditional can be considered unnecessary logic (feel free to discuss further in [#8179](https://github.com/medic/cht-core/issues/8179#issuecomment-1682863877))